src/ansys/dynamicreporting/core/utils/report_remote_server.py

```diff
@@ -104,6 +104,13 @@ class Server:
     """
 
     def __init__(self, url=None, username=None, password=None):
+        # Check on the validity of url formatting
+        if url is not None:
+            o = urlparse(url)
+            if o is False:
```

Comment:
@margalva I'm not sure if this will return a bool. Probably just a named tuple? https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse

Reply:
@viseshrp yeah fat fingered it. Thanks for catching it. Should be fixed now.